e2e/cypress/integration/enterprise/cloud/billing/subscriptions_free_trial_spec.js

```diff
@@ -0,0 +1,204 @@
+
+        // * Check for Subscription header
+        cy.contains('.admin-console__header', 'Subscription').should('be.visible');
+    });
+
+    beforeEach(() => {
+        // # Click on close button of subscribe window if exist
+        cy.get('body').then(($body) => {
+            if ($body.find('#closeIcon').length > 0) {
+                cy.get('#closeIcon').parent().click();
+            }
+        });
+    });
+
+    it('MM-T4118 Subscription page UI check', () => {
+        // * Check for visibility of Trial tag
+        cy.contains('span', 'Trial', {timeout: 10000}).should('be.visible');
+
+        // * Check for User count
+        cy.request('/api/v4/analytics/old?name=standard&team_id=').then((response) => {
+            cy.get('.PlanDetails__userCount > span').invoke('text').then((text) => {
+                const userCount = response.body.find((obj) => obj.name === 'unique_user_count');
+                expect(text).to.contain(userCount.value);
+            });
+        });
+
+        // * Check for Contact Sales navigation
+        const email = encodeURIComponent(adminUser.email);
+        const name = encodeURIComponent(`${adminUser.first_name} ${adminUser.last_name}`);
+        cy.contains('span', 'Contact Sales').parent().then((link) => {
+            const getHref = () => link.prop('href');
+            cy.wrap({href: getHref}).invoke('href').should('contains', `/contact-us?email=${email}&name=${name}&inquiry=sales&inquiry-issue=trial_questions`);
+            cy.wrap(link).should('have.attr', 'target', '_blank');
+            cy.wrap(link).should('have.attr', 'rel', 'noopener noreferrer');
+            cy.request(link.prop('href')).its('status').should('eq', 200);
+        });
+
+        // * Check for See how billing works navigation
+        cy.contains('span', 'See how billing works').parent().then((link) => {
+            const getHref = () => link.prop('href');
+            cy.wrap({href: getHref}).invoke('href').should('contains', '/cloud-billing.html');
+            cy.wrap(link).should('have.attr', 'target', '_new');
+            cy.wrap(link).should('have.attr', 'rel', 'noopener noreferrer');
+            cy.request(link.prop('href')).its('status').should('eq', 200);
+        });
+    });
+
+    it('MM-T4122 "Subscribe now" navigation and closing of Subscribe window', () => {
+        // # Click on Subscribe Now button
+        cy.contains('span', 'Subscribe Now').parent().click();
+
+        // * Check for "Provide Your Payment Details" label
+        cy.get('.title').contains('span', 'Provide Your Payment Details').should('be.visible');
+
+        // # Click on close button of subscribe window
+        cy.get('#closeIcon').parent().should('exist').click();
+
+        // * Check for "You're currently on a free trial" label
+        cy.contains('span', "You're currently on a free trial").should('be.visible');
+    });
+
+    it('MM-T4123 Purchase modal UI check', () => {
```

Comment:
I messed up archiving Zephyr test case. This should now be `MM-T4124`.
```suggestion
    it('MM-T4124 Purchase modal UI check', () => {
```